package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Cell Tower Coverage",
+  "version": "0.1.1",
+  "description": "A starting point for new projects and atlas.utah.gov",
+  "homepage": "https://atlas.utah.gov",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/agrc/AGRCJavaScriptProjectBoilerPlate.git"
+  },
```

Comment:
i would guess `http://mapserv.utah.gov/celltowercoverage`. Anything other than atlas makes sense at this point.